frontend-react/src/App.tsx

```diff
@@ -112,166 +115,184 @@ const App = () => {
     });
 
     if (isIE) return <ErrorPage code={CODES.UNSUPPORTED_BROWSER} />;
+
     return (
         <Security
             oktaAuth={OKTA_AUTH}
             onAuthRequired={customAuthHandler}
             restoreOriginalUri={restoreOriginalUri}
         >
             <SessionProvider oktaHook={useOktaAuth}>
-                <Suspense fallback={<Spinner size={"fullpage"} />}>
-                    <NetworkErrorBoundary
-                        fallbackComponent={() => <ErrorPage type="page" />}
-                    >
-                        <DAPHeader
-                            env={process.env.REACT_APP_ENV?.toString()}
-                        />
-                        <GovBanner aria-label="Official government website" />
-                        <SenderModeBanner />
-                        <ReportStreamHeader />
-                        {/* Changed from main to div to fix weird padding issue at the top
+                <QueryClientProvider client={queryClient}>
```

Comment:
all of the below changes are just whitespace changes as a result of adding the QueryClientProvider into the heirarchy.

Note: it might be nice to split all of our providers out into an app wrapper component so we don't get this huge diff every time we update this file...